### PR TITLE
Support ruby 1.9.3

### PR DESCRIPTION
**Pull request by jabley**
Despite 1.9 being EOL, some people still need it. This changes adds
support for that. The main issue was __dir__ is a 2.x feature.

### CODE REVIEW

**Comment by jabley** [comment]
OK, so it's not as straightforward as I'd hoped. Would you be interested in this change if I can get it working?

**Comment by jabley** [comment]
I have the tests passing, but I'm not sure what I've done is correct. [The ruby issue that introduced `rb_thread_call_without_gvl`](https://bugs.ruby-lang.org/issues/4328) talks about it being in `internal.h` and attempts to use it cause a compiler warning.

File: ext/brotli/extconf.rb [hunk 1]
```diff
@@ -7,12 +7,18 @@
 create_makefile('brotli/brotli')
 
 def acopy(dir)
+  # __dir__ is only supported >= ruby 2.0, use __FILE__ approach
+  # Gem should support wide range of ruby versions
+  def _dir
+    File.dirname(__FILE__)
+  end
```
Comment (miyucy) [inline ext/brotli/extconf.rb]: I think this func does not need. just write `File.dirname(__FILE__)` as like spec file.

**Comment by miyucy** [comment]
> I have the tests passing, but I'm not sure what I've done is correct. The ruby issue that introduced rb_thread_call_without_gvl talks about it being in internal.h and attempts to use it cause a compiler warning.

Oh, I'm don't know too. It scary...
I want do not give `without_gvl` feature to Ruby v1.9.  Because, 1.9 is EOL (as you know)  so if strange behavier (or uncaught bug) was found anyone can not fix it.
So, I want to safty. What do you think?

**Comment by jabley** [comment]
I think not supporting 1.9 is a reasonable position, since it's been EOL for a number of years. People that still have to use 1.9 should be encouraged to upgrade.

**Comment by miyucy** [comment]
Ah, sorry for Misleading response.
I want to support 1.9 as possible.
But, I wantn't to introduce `without_gvl` function in 1.9.
How about this? https://github.com/jabley/brotli/pull/1

**Pull request merged**